src/assets/scripts/client/ui/TutorialView.js

```diff
@@ -450,33 +438,32 @@ export default class TutorialView {
 
         this.tutorial_step({
             title: 'Bon voyage, aircraft!',
-            text: ['When the aircraft crosses the airspace boundary, it will ',
+            text: ['When the aircraft leaves your airspace , it will switch to center and',
                    'automatically remove itself from the flight strip bay on the right.',
-                   'Congratulations, you\'ve successfully taken off one aircraft.'
+                   'Congratulations, you\'ve successfully departed an aircraft.'
             ].join(' '),
             side: 'left',
             position: tutorial_position
         });
 
         this.tutorial_step({
             title: 'Arrivals',
-            text: ['Now, onto arrivals. Click on any arriving aircraft in the radar screen; after',
-                   'you\'ve selected it, use the altitude/heading/speed controls you\'ve learned in',
-                   'order to guide it to be in front of a runway. Make sure to get the aircraft down to',
-                   'around 4,000ft, and 10-15 nautical miles (2-3 range rings) away from the airport.',
-                   'While you work the airplane, read the next step.'
+            text: ['Now, onto arrivals. Click on any arriving aircraft in the radar screen; after you\'ve',
+                   'selected it, use the altitude/heading/speed controls you\'ve learned in order to',
+                   'guide it to the intercept of the ILS for the runway. The aircraft should be at an',
+                   'appropriate altitude and flying an appropriate heading before you clear it for approach!'
```

Comment:
We could give some indication of what 'appropriate' altitudes and headings are

Reply:
I see you're clarifying what is mean here later on, but it would be good to say something like "more on this later"

Reply:
Sure no problem, I'll get that sorted!